### DISCUSSION TITLE
fluentui-react docs webpage error

### DISCUSSION THREAD

**Discussion by DicoBarbosa** [microsoft/fluentui#17355] [Q&A] [answered]
Having trouble accessing the Docs webpage for FluentUI React Project [https://aka.ms/fluentui-react].
Anyone else with the same problem?

**Comment by khmakoto** [comment, marked as answer]
It seems to be working for me, can you check again @DicoBarbosa?

**Reply by DicoBarbosa** [reply]
Now it's working for me too. I don't know what happened.
Anyways tks @khmakoto
